src/main/play-doc/operation/Install.md

```diff
@@ -255,19 +259,112 @@ On some Debian distributions you may need to add a dedicated Personal Package Ar
 
 ConductR provides a ConductR-HAProxy bundle that listens for bundle events from ConductR and updates the local HAProxy configuration file accordingly. We must specifically allow the bundle to use `sudo` to reload HAProxy.
 
-First, we have the user `conductr-agent` own the HAProxy config file.
+We have the user `conductr-agent` own the HAProxy config file.
+
+```bash
+[172.17.0.1]$ sudo chown conductr-agent:conductr-agent /etc/haproxy/haproxy.cfg
+```
+
+### Installing Docker based HAProxy
+
+We will use HAProxy version 1.5 official Docker image published to [Docker Hub](https://hub.docker.com/_/haproxy/).
+
+#### Prerequisite
+
+Docker needs to be installed on the host where HAProxy docker container will run.
+
+#### Preparing HAProxy config
+
+Setup basic HAProxy configuration for the Docker based HAProxy. Ensure the `conductr-agent` user which runs the ConductR-HAProxy has ownership to the HAProxy config file.
 
 ```bash
+[172.17.0.1]$ sudo mkdir /etc/haproxy
+[172.17.0.1]$ sudo touch /etc/haproxy/haproxy.cfg
 [172.17.0.1]$ sudo chown conductr-agent:conductr-agent /etc/haproxy/haproxy.cfg
 ```
 
+Populate the `/etc/haproxy/haproxy.cfg` file with the following configuration.
+
+```
+defaults
+    log global
+    mode    http
+    option  httplog
+    option  dontlognull
+    timeout connect 5000
+    timeout client  50000
+    timeout server  50000
+
+frontend monitor
+  bind :65535
+  mode http
+  monitor-uri /test
+```
```

Comment:
We certainly need to comment on the opening. What is it for? I should like to know more about this before allowing it.